src/main/java/spireMapOverhaul/zones/brokenSpace/relics/BrokenPrayerWheel.java

```diff
@@ -0,0 +1,59 @@
+package spireMapOverhaul.zones.brokenSpace.relics;
+
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import com.megacrit.cardcrawl.relics.PrayerWheel;
+import com.megacrit.cardcrawl.rewards.RewardItem;
+import javassist.CtBehavior;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+
+public class BrokenPrayerWheel extends BrokenRelic {
+    public static final String ID = "BrokenPrayerWheel";
+    public static final int GOLD = 2;
+    public static final int AMOUNT = 1;
+
+
+    public BrokenPrayerWheel() {
+        super(ID, RelicTier.SPECIAL, LandingSound.CLINK, PrayerWheel.ID);
+
+    }
+
+    @Override
+    public int changeNumberOfCardsInReward(int numberOfCards) {
+        return numberOfCards - AMOUNT;
+    }
+
+    @Override
+    public String getUpdatedDescription() {
+        return DESCRIPTIONS[0] + AMOUNT + DESCRIPTIONS[1] + GOLD + DESCRIPTIONS[2];
+    }
+
+    @SpirePatch2(clz = RewardItem.class, method = "applyGoldBonus")
+
+    public static class BrokenPrayerWheelPatch {
+        @SpireInsertPatch(locator = Locator.class, localvars = {"tmp"})
+        public static void patch(RewardItem __instance, int tmp) {
+            if (AbstractDungeon.player.hasRelic(makeID(ID))) {
+                int prayerWheelAmt = 0;
+                for (AbstractRelic r : AbstractDungeon.player.relics) {
+                    if (r instanceof BrokenPrayerWheel) {
+                        prayerWheelAmt++;
+                    }
+                }
+
+                __instance.bonusGold += (int) (tmp * Math.pow(GOLD, prayerWheelAmt));
```

Comment:
Using `Math.pow` here isn't seem right. If you had two copies of this relic. it would square the amount of gold. Three would cube it. Even at one copy I don't think this is right; if you get 100 gold, this will give you 200 bonus gold for a total of 300 gold.

Having multiple copies of a relic is a very niche interaction, but the simplest way to handle this is to avoid exponential scaling at all. Treat each copy of the relic as an additive increase (e.g. +100% gold), and write this as:
```suggestion
                __instance.bonusGold += (int) (tmp * prayerWheelAmount * (GOLD - 1)));
```

Reply:
yeah, that makes sense. I totally misunderstood what that function was doing, Thanks!